### DISCUSSION TITLE
what is html[data-theme] for?

### DISCUSSION THREAD

**Discussion by MoofyWoofy** [smaranjitghose/awesome-portfolio-websites#1239] [Q&A] [answered]
What is the purpose **html[data-theme]** in the CSS files? As I notice that it uses **html[light-mode]** to manage the theme of the website.

**Comment by anushbhatia** [comment, marked as answer]
It is used to specify the colors of various elements in dark and light mode. for example a button in dark mode might be white but in light mode it might be black so to specify that we use data-theme
